### DISCUSSION TITLE
Keep comments after issue moved to discussion

### DISCUSSION THREAD

**Discussion by Anilople** [community/community#3055] [Discussions]
When someone move repo's issue to discussion, the comment in issue will lose.

Is there any method to keep them?

There are many useful comment in origin issue.

Thanks

**Comment by becca** [comment]
This is a bug and should no longer be happening. We fixed this yesterday. 
When did you convert an issue to a discussion and saw this occur?
Is there a specific issue you have which lost comments? If so, please share and we'll do our best to restore those.

**Reply by Anilople** [reply]
Thanks.

Just a few comments, already add it manually.

The bug still exists.

Newest Discussion here ctripcorp/apollo/discussions/3363 

There are comments losed.

**Reply by becca** [reply]
We're looking into this and hope to recover the lost comments from the latter listed issue//discussion. I'll keep you posted.
